cartography/intel/gcp/storage.py

```diff
@@ -0,0 +1,158 @@
+        bucket['log_bucket'] = b.get('logging', {}).get('logBucket')
+        bucket['requester_pays'] = b.get('billing', {}).get('requesterPays', None)
+        bucket_list.append(bucket)
+    return bucket_list
+
+
+def load_gcp_buckets(neo4j_session, buckets, gcp_update_tag):
+    '''
+    Ingest GCP Storage Buckets to Neo4j
+
+    :type neo4j_session: Neo4j session object
+    :param neo4j session: The Neo4j session object
+
+    :type buckets: list
+    :param buckets: List of GCP Buckets to injest
+
+    :type gcp_update_tag: timestamp
+    :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+
+    :rtype: NoneType
+    :return: Nothing
+    '''
+
+    query = """
+    MERGE(p:GCPProject{projectnumber:{ProjectNumber}})
+    ON CREATE SET p.firstseen = timestamp()
+    SET p.lastupdated = {gcp_update_tag}
+
+    MERGE(bucket:GCPBucket{id:{BucketId}})
+    ON CREATE SET bucket.firstseen = timestamp(),
+    bucket.bucket_id = {BucketId}
+    SET bucket.self_link = {SelfLink},
+    bucket.project_number = {ProjectNumber},
+    bucket.kind = {Kind},
+    bucket.location = {Location},
+    bucket.location_type = {LocationType},
+    bucket.labels = {Labels},
+    bucket.meta_generation = {MetaGeneration},
+    bucket.storage_class = {StorageClass},
+    bucket.time_created = {TimeCreated},
+    bucket.retention_period = {RetentionPeriod},
+    bucket.iam_config_bucket_policy_only = {IamConfigBucketPolicyOnly},
+    bucket.owner_entity = {OwnerEntity},
+    bucket.owner_entity_id = {OwnerEntityId},
+    bucket.lastupdated = {gcp_update_tag},
+    bucket.versioning_enabled = {VersioningEnabled},
+    bucket.log_bucket = {LogBucket},
+    bucket.requester_pays = {RequesterPays},
+    bucket.default_kms_key_name = {DefaultKmsKeyName}
+
+    MERGE (p)-[r:RESOURCE]->(bucket)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {gcp_update_tag}
+    """
+    for bucket in buckets:
+        neo4j_session.run(
+            query,
+            ProjectNumber=bucket['project_number'],
+            BucketId=bucket['id'],
+            SelfLink=bucket['self_link'],
+            Labels=bucket['labels'],
+            Kind=bucket['kind'],
+            Location=bucket['location'],
+            LocationType=bucket['location_type'],
+            MetaGeneration=bucket['meta_generation'],
+            StorageClass=bucket['storage_class'],
+            TimeCreated=bucket['time_created'],
+            RetentionPeriod=bucket['retention_period'],
+            IamConfigBucketPolicyOnly=bucket['iam_config_bucket_policy_only'],
+            OwnerEntity=bucket['owner_entity'],
+            OwnerEntityId=bucket['owner_entity_id'],
+            VersioningEnabled=bucket['versioning_enabled'],
+            LogBucket=bucket['log_bucket'],
+            RequesterPays=bucket['requester_pays'],
+            DefaultKmsKeyName=bucket['default_kms_key_name'],
+            gcp_update_tag=gcp_update_tag,
+        )
```

Comment:
Ah, we're super close but I totally missed this: we need a `sync_buckets()` function which will call your `get` + `transform` + `load`.

Then we need to call `sync` here: https://github.com/lyft/cartography/blob/6e060389fbeb14f4ccc3e58005230129f1c6962f/cartography/intel/gcp/__init__.py#L61

Reply:
My bad, I should have noticed, thanks!

Reply:
Hm, for some reason it's not passing Travis somehow despite running `make test_lint`